keda.yaml

```diff
@@ -10689,6 +10710,9 @@ spec:
         app.kubernetes.io/part-of: keda-operator
         app.kubernetes.io/version: 2.16.1
         app.kubernetes.io/instance: keda
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8080"
```

Comment:
I wonder why `prometheus.io/path` is missing in this case

Reply:
this is rendered by keda's upstream charts, right?
Could it be because keda's metric server does not have a dedicated path `/metrics` as this is the workloads root path?

Reply:
`this is rendered by keda's upstream charts, right?`
right

`Could it be because keda's metric server does not have a dedicated path /metrics as this is the workloads root path?`
I'm not sure, maybe